### ISSUE TITLE
Mod affects the "reset vehicle camera" when it's set to off. 

### ISSUE THREAD

**Issue by BainrowKiss** [loki79uk/FS22_CabView#8]
Great mod, I really enjoy having this one. 

My only gripe is that the vehicle camera resets every time I exit the vehicle, no matter if I set the "reset vehicle camera" to on or off. 

Removing your mod solves the issue, but I refuse to play without it, again; great mod! (I have another bug where the boom on wheel and skid loaders doesn't work after moving the camera, so I can't set the camera to a preferred position without removing your mod when operating these vehicles. Bit tiresome either way, but I've sent in a bug report to Giants about this issue.) 

If you don't mind, I have a small wish for your mod (Although I think it's a shame that Giants doesn't offer this in their simulator games, I think I have a bigger chance of getting this implemented by sharing the idea with you instead, and hope that you like it enough to spend your time adding it to your mod): **Basically: Adjust the camera height in settings** (for individual vehicles preferably, but most cameras are set too high, so a universal setting would be welcomed as well).

The new Volvo 380DL excavator has just about the same view when leaning forward, and doesn't remove the blind spot caused by the huge metal beam across the top of the front window. I think if I'd be able to adjust the height lower, it would help a lot. (I also think the height in most trucks are way too high, and it's pretty evident when sitting inside the new skidder (JD 848L-II) as well.

Appreciate the work you've done with this mod! Hope you find the inspiration to do some more.. :)

**Comment by loki79uk** [comment]
Hi - I'm glad that you like it.

This is an instance where it would be good to have some settings/options.

The resetting behaviour is intentional, because it annoyed me how the last known camera position would persist at all times.  I think whenever I switch cameras for the current vehicle then yes I want it to remember the settings for each camera, BUT if I exit to drive another vehicle, sleep, go to the shop, etc, then when I return it should be reset as if I got in for the first time.  Why should the game remember that the last time I drive a vehicle (e.g. yesterday) that I was looking over my shoulder just before I got out.

**Comment by BainrowKiss** [comment]
Hi,
Thanks for the replies and alternative version, but unfortunately this version only saves the camera for up/down, and not left/right. Would it be possible to sort this out? Would've been fine as it is, but several vehicles, like the Volvo 380 and Claas tractor ++, have the default camera set a bit to either side. 

I'm guessing your mod overwrites the vanilla "vehicle camera reset" on/off setting completely? 
Can see it gets annoying when it remembers the camera angle hours or days later.. Would be nice if the camera would reset after a set time or after you've entered another vehicle. 

I haven't tested out the Autoload, but might check it out at some point. I see it's popular, so that's cool! 
I'm really happy you've put the "per vehicle height settings" to your list, that's all I was hoping for. Thanks a lot, looking forward to see this implemented at some point!

**Comment by loki79uk** [comment]
That is weird. I'm sure I tested it by tabbing around different vehicles and it was keeping the same position for each one.  Or do you mean after a save and reload?  I did not test it with that.

**Comment by BainrowKiss** [comment]
Ok, yeah, it's on my end, my bad. Tried with only your mod activated and it worked fine.
I'll run through my mod list and see what's causing the issue. 

Thanks again for this version, this is so awesome. Fixed my problem faster than Giants ever could! :D

**Comment by loki79uk** [comment]
Glad it works.  Do you have VCA?  I think that also affects the camera, and the two mods have conflicted in the past.  I think there was a setting you can turn off for that, but could easily be a different mod.

**Comment by BainrowKiss** [comment]
Yeah, it's VCA. I've got all camera options set to off in both global and local settings for the mod. Don't know what else could affect it, or would you happen to know which setting it could be if there is one? 

Don't know which one of you two would have the least work of trouble shooting this conflict, but I could perhaps ask Mogli12 who has the VCA if he can take a look at it? Is is perhaps a bit silly to hope that these two mods would be conflict free? 

Was thinking I had to chose between the two mods, but I've played with both activated last couple of hundreds of hours, so I'll just keep using them both I think. VCA is such a game changer for analogue controllers, so it's a must-have mod in my case. 
Can't really imagine playing without the ability to lean forward either now. :)
